src/mavlink-router/endpoint.cpp

```diff
@@ -427,6 +428,37 @@ void Endpoint::log_aggregate(unsigned int interval_sec)
     }
 }
 
+static bool _expired(void *data)
+{
+    log_warning("Endpoint expired.");
+    Mainloop::get_instance().remove_dynamic_endpoint(static_cast<Endpoint *>(data));
+    return true;
+}
+
+#define EXPIRE_TIMEOUT 30 * MSEC_PER_SEC
```

Comment:
`EXPIRE_TIMEOUT_MSEC`
Could this be constexpr?

Reply:
Yes, changed to constexpr.